### ISSUE TITLE
error on: npm install -g xsd2json

### ISSUE THREAD

**Issue by gabrielmancini** [fnogatz/xsd2json#40]
```
npm install -g xsd2json
npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/gabrielmancini/.nvm/versions/node/v0.12.4/bin/node" "/Users/gabrielmancini/.nvm/versions/node/v0.12.4/bin/npm" "install" "-g" "xsd2json"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! path /Users/gabrielmancini/.nvm/versions/node/v0.12.4/lib/node_modules/xsd2json/lib-pl/cli
npm ERR! code ENOENT
npm ERR! errno -2

npm ERR! enoent ENOENT, chmod '/Users/gabrielmancini/.nvm/versions/node/v0.12.4/lib/node_modules/xsd2json/lib-pl/cli'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/gabrielmancini/Developer/Watson/jsonMass/schema/npm-debug.log
```

**Comment by fnogatz** [comment]
Thank you for this bug report. The error occurred because `npm install` first tries to link the `bin` file specified in the `package.json`, which is created by xsd2json just in the `postinstall` step. I added a placeholder file which is replaced after installation. The fixed version is available at npm as `v1.7.6`.

**Comment by mellis481** [comment]
I get this same error for v1.12.1...

```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! xsd2json@1.12.2 preinstall: `npm run create-qlf`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the xsd2json@1.12.2 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
